src/main/java/auto_batcher/Main.java

```diff
@@ -25,23 +25,29 @@ public static void main(String[] args) {
 				multiStudentInvoker = new MultiStudentScheduleInvoker(invoker);
 			}
 
+			ArrayList<Student> groupedStudents = new ArrayList();
```

Comment:
i would have preffered this being null. and then doing a check for null. on line 47. but this is not a deal breaker